components/shared_code/src/shared_data_model/sources/owasp_dependency_check.py

```diff
@@ -30,6 +30,7 @@
     name="OWASP Dependency-Check",
     description="OWASP Dependency-Check is a utility that identifies project dependencies and checks if there are any "
     "known, publicly disclosed, vulnerabilities.",
+    supported_versions="Versions that use v2.0 - v2.5, v3.0, v3.1, or v4.0 of the OWASP Dependency-Check XSD",
```

Comment:
It doesn't seem to be super easy to find out which version is being used from the url below, nor search engine. Is it referenced in the source output data somewhere?

Reply:
I have changed it to `>=9.0, <=9.2`. Versions older than 9.0 are not supported by OWASP because of the source database API changes. I did not remove the old XSD versions so Quality-time will still work with older versions, but they shouldn't be used anymore.